### ISSUE TITLE
In Explore, dates from ES datasource show as invalid

### ISSUE THREAD

**Issue by ChadiEM** [grafana/grafana#20222]
**What happened**:
Dates are showing up as "Invalid date" in Explore view.

**What you expected to happen**:
We expect to see the actual datetimes.

**How to reproduce it (as minimally and precisely as possible)**:
Configure an ES datasource with:
index = [event-]YYYY.MM.DD
pattern = daily
version = 5.x
time field = timestamp

**Anything else we need to know?**:
Was working in 6.3.x (can double check which patch version if needed)

msearch response:
```
{"responses":[{"took":3,"timed_out":false,"_shards":{"total":5,"successful":5,"failed":0},"hits":{"total":1,"max_score":null,"hits":[{"_index":"event-2019.11.06","_type":"log_type","_id":"AW5C9-VTpjD2ExF25OY0","_score":null,"_source":{"type":"json","body":{"t":"EVT","v":"1.0","evt":{"c":"SFK-07","level":"INFO","dsc":"Some message"}},"timestamp":"2019-11-06T23:04:56.643Z"},"fields":{"timestamp":[1573081496643]},"sort":[1573081496643]}]},"aggregations":{"2":{....}},"status":200}]}
```

**Environment**:
- Grafana version: Reproduced on 6.4.3, 6.4.4 and 6.5.0 (nightly)
- Data source type & version: Elasticsearch 5.0.1
- OS Grafana is installed on: RHEL 7.2
- User OS & Browser: Windows 10 / Chrome 78 & Firefox 70
- Grafana plugins:
- Others:

**Comment by torkelo** [comment]
dupe of https://github.com/grafana/grafana/issues/19909, fix coming in 6.5

**Comment by ChadiEM** [comment]
Hey @torkelo, not exactly. They show up as `Invalid date`, unlike the issue you mentioned where they show up as `2019-01-01 00:00:00`. I also tried on the latest docker image and was able to reproduce.
